### PR TITLE
[ER]: route tables

### PR DESCRIPTION
**Pull request by artem-lifshits**
### Acceptance tests
```
=== RUN   TestRouteTableLifeCycle
    route_tables_test.go:39: Attempting to create enterprise router
    route_tables_test.go:70: Attempting to create route table
    route_tables_test.go:86: Attempting to update route table
    route_tables_test.go:95: Attempting to list route table
    tools.go:72: {
          "route_tables": [
            {
              "id": "42b73a31-cd42-4737-89e4-6fa701edc360",
              "name": "acctest_route-table-OraR-updated",
              "description": "test vpc attachment",
              "is_default_association": false,
              "state": "available",
              "tags": [
                {
                  "key": "muh",
                  "value": "muh"
                },
                {
                  "key": "test",
                  "value": "test"
                }
              ],
              "bgp_options": {
                "load_balancing_as_path_ignore": false,
                "load_balancing_as_path_relax": false
              },
              "created_at": "2024-07-10T11:54:27.221Z",
              "updated_at": "2024-07-10T11:54:31.58Z"
            }
          ],
          "page_info": {
            "next_marker": "",
            "current_count": 1
          },
          "request_id": "9f68a54e2f7ff64de9cb2a4bae6c2951"
        }
    route_tables_test.go:80: Attempting to delete route table
    route_tables_test.go:48: Attempting to delete enterprise router
--- PASS: TestRouteTableLifeCycle (75.12s)
PASS

Process finished with the exit code 0
```

### CODE REVIEW

File: acceptance/openstack/er/route_tables_test.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+	t.Logf("Attempting to create enterprise router")
+
+	createResp, err := instance.Create(client, createOpts)
+	th.AssertNoErr(t, err)
+
+	err = waitForInstanceAvailable(client, 100, createResp.Instance.ID)
+	th.AssertNoErr(t, err)
+
+	defer func(client *golangsdk.ServiceClient, id string) {
```
Comment (anton-sidelnikov) [inline acceptance/openstack/er/route_tables_test.go]: why not t.Cleanup?

File: acceptance/openstack/er/route_tables_test.go [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+	t.Logf("Attempting to create route table")
+	createRtResp, err := route_table.Create(client, createRouteTableOpts)
+	th.AssertNoErr(t, err)
+	th.AssertEquals(t, createRouteTableOpts.Name, createRtResp.Name)
+	th.AssertEquals(t, *createRouteTableOpts.Description, createRtResp.Description)
+
+	err = waitForRouteTableAvailable(client, 300, createResp.Instance.ID, createRtResp.ID)
+	th.AssertNoErr(t, err)
+
+	defer func(client *golangsdk.ServiceClient, erId, rtId string) {
```
Comment (anton-sidelnikov) [inline acceptance/openstack/er/route_tables_test.go]: Cleanup?

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: acceptance/openstack/er/route_tables_test.go (hunk 1 above)
Comment (artem-lifshits) [reply acceptance/openstack/er/route_tables_test.go]: fixed

File: acceptance/openstack/er/route_tables_test.go (hunk 2 above)
Comment (artem-lifshits) [reply acceptance/openstack/er/route_tables_test.go]: fixed

**Review by anton-sidelnikov — APPROVED** [review]

**Review by sgmv — APPROVED** [review]

**Pull request merged**